spec/models/education_benefits_claim_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+RSpec.describe EducationBenefitsClaim, type: :model do
+  let(:attributes) do
+    {
+      form: { foo: true }
+    }
+  end
+  subject { described_class.new(attributes) }
+
+  describe "validations" do
+    it { should validate_presence_of(:form) }
```

Comment:
@saneshark if we do it this way we also have to make sure all of subject's other fields are valid which we shouldnt need to care about to test the form field. from looking at the shoulda code it's setting a field to a value, running validations, then seeing if error messages for the field includes the error it wants to see. that seems like a real unit test and not too fragile, what don't you like about their method?